### PR TITLE
[Improve] check permissions improvement

### PR DESCRIPTION
**Pull request by wolfboys**
[Improve] check permissions improvement

### CODE REVIEW

**Comment by wolfboys** [comment]
@zhoulii PTAL. thx

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/exception/ApiAlertException.java [hunk 1]
```diff
@@ -57,4 +57,10 @@ public static void throwIfFalse(boolean expression, String errorMessage) {
       throw new ApiAlertException(errorMessage);
     }
   }
+
+  public static void throwIfTrue(boolean expression, String errorMessage) {
```
Comment (zhoulii) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/exception/ApiAlertException.java:61]: Great, we really need this method,

**Review by zhoulii — APPROVED** [review]
LGTM. Nice work, it makes code more readable.

**Review by muchunjin — APPROVED** [review]
LGTM

**Pull request merged**